src/domain/hanke/list/Hankelista.styles.scss

```diff
@@ -0,0 +1,74 @@
+.hankelista {
+  padding: 0 var(--spacing-l) var(--spacing-3-xl) var(--spacing-l);
+
+  &__inner {
+    padding: var(--spacing-l) var(--spacing-l) var(--spacing-3-xl) var(--spacing-l);
+    background: var(--color-white);
+
+    &__buttonWpr {
```

Comment:
I think we should avoid 3rd level nesting if possible. So buttonWpr and hankeLink should be move to upper level (ie. hankeLista__inner, hankeLista__buttonWpr)